File: 241128/소수 판별기/prime-number-discriminator.py
n = int(input())

satisfied = True #응 소수야! 약수가 1이랑 나 자신밖에 없어 -> 나중에 아니면 False

for i in range(2, n):
    if (n%i==0) :
        satisfied = False #응 나 소수 아니야

if satisfied == True :
    print("P")
else :
    print("C")

File: 241128/소수 판별기/prime-number-discriminator.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    // 여기에 코드를 작성해주세요.

    int n;

    scanf("%d", &n);

    bool satisfied = true; //응 나 소수야! 

    for (int i=2; i<n; i++){
        if (n%i==0) {
            bool satisfied = false; //응 ㅋㅋ 나 소수 아니야
        }
    }

    if (satisfied == true) {
        printf("P");
    }
    else {
        printf("C");
    }
    return 0;
}